app/views/transaction/_additional_information_tabbed.html.erb

```diff
@@ -1,4 +1,3 @@
-<div data-module="track-start-page-tabs">
   <%
```

Comment:
Nitpick: now that the wrapping DIV is gone, should this whole file be un-indented?

Reply:
@andysellick Thanks, done - good spot